src/tests/ftest/aggregation/shutdown_restart.py

```diff
@@ -24,7 +24,7 @@ def get_nvme_free_space(self):
         return free_space
 
     def test_ioaggregation(self):
-        """Jira ID: DAOS-4332.
+        """Jira ID: DAOS-3752.
 
```

Comment:
DAOS-8677 wasn't on this PR?
Seems only comment changed.

Reply:
Yes, just a comment change.